R/segmentBins.R

```diff
@@ -125,21 +119,13 @@ setMethod("segmentBins", signature=c(object="QDNAseqCopyNumbers"),
         ## use sample names for indexing, they are available in the CNA objects
         ## as the name of the third column
         names(msgs) <- sampleNames(object)
-        if ("parallel" %in% .packages(all.available=TRUE) && mc.cores > 1) {
-            segments <- parallel::mclapply(cna, FUN=function(x, ...) {
-                vmsg(msgs[colnames(x)[3]])
-                segment(x, alpha=alpha, undo.splits=undo.splits,
-                    undo.SD=undo.SD, verbose=0, ...)
-            }, ..., mc.cores=mc.cores)
-        } else {
-            segments <- lapply(cna, FUN=function(x, ...) {
-                vmsg(msgs[colnames(x)[3]], appendLF=FALSE)
-                seg <- segment(x, alpha=alpha, undo.splits=undo.splits,
-                    undo.SD=undo.SD, verbose=0, ...)
-                vmsg()
-                seg
-            }, ...)
-        }
+        segments <- flapply(cna, FUN=function(x, ...) {
+            vmsg(msgs[colnames(x)[3]], appendLF=FALSE)
```

Comment:
There's actually a small difference in the `mclapply()` and `lapply()` versions I wrote here: The first one doesn't include a `vmsg(..., appendLF=FALSE)`, so that all the parallel messages don't end up the same line (and the corresponding empty `vmsg()` afterwards). Could you please use that one instead.